modules/programs/zsh/prezto.nix

```diff
@@ -352,15 +352,15 @@ in {
   };
   config = mkIf cfg.enable (mkMerge [{
     home.file."${relToDotDir ".zprofile"}".text =
-      builtins.readFile "${pkgs.zsh-prezto}/runcoms/zprofile";
+      builtins.readFile "${pkgs.zsh-prezto}/share/zsh-prezto/runcoms/zprofile";
     home.file."${relToDotDir ".zlogin"}".text =
-      builtins.readFile "${pkgs.zsh-prezto}/runcoms/zlogin";
+      builtins.readFile "${pkgs.zsh-prezto}/share/zsh-prezto/runcoms/zlogin";
     home.file."${relToDotDir ".zlogout"}".text =
-      builtins.readFile "${pkgs.zsh-prezto}/runcoms/zlogout";
+      builtins.readFile "${pkgs.zsh-prezto}/share/zsh-prezto/runcoms/zlogout";
     home.packages = with pkgs; [ zsh-prezto ];
 
     home.file."${relToDotDir ".zshenv"}".text =
-      (builtins.readFile "${pkgs.zsh-prezto}/runcoms/zshenv");
+      (builtins.readFile "${pkgs.zsh-prezto}/share/zsh-prezto/runcoms/zshenv");
```

Comment:
Why not replace the `.text` for `.source` at the same time?

Like `home.file."${relToDotDir ".zprofile"}".source = "${pkgs.zsh-prezto}/share/zsh-prezto/runcoms/zprofile";`

Reply:
done in latest commit